.gitignore

```diff
@@ -20,7 +20,7 @@ node_modules
 packages/api/dist
 packages/api/@types
 packages/crdt/dist
-packages/desktop-electron/client-build
+packages/desktop-electron/build
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏would you mind keeping `client-build` in here too? I don't think `git pull` would delete the existing `client-build` folders for folks. So people might start accidentally committing them.

Reply:
updated 👍